Shubham_Sharma/JWT Assignment/UserAuthorization/UserAuthorization/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace UserAuthorization.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ValuesController : ControllerBase
+    {
+        // GET api/values
+        [HttpGet]
+        public ActionResult<IEnumerable<string>> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/values/5
+        [HttpGet("{id}")]
```

Comment:
remove unused controller

Reply:
Then what should I select for the default controller ? Because when we run the IIS Express,  then by default it opens the browser for values controller.

Reply:
Can we Not Run the Application without a default Controller being Set?

Reply:
We can run the application without that, but just for checking that whether our service is running or not, I am using this.